composer.json

```diff
@@ -40,31 +40,33 @@
         "drupal/config_split": "^1.1",
         "drupal/console": "~1.0",
         "drupal/contact_storage": "1.0.0-beta8",
-        "drupal/core": "8.4.*",
+        "drupal/core": "8.5.0-beta1",
```

Comment:
Hm. I guess we should probably start all new projects on this Beta, if we start them soon, but I also want to make sure we don't get stuck at the Beta. Does `8.5.x` not work here?